board/flash_gen3.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+set -e
+
+PANDA_GEN3=1 scons -u
```

Comment:
discussed this as well, let's use `PANDA_H7` instead of the `GEN3`. `GEN2` implies the use of OBD-C